hybrid/layout/pv_design_utils.py

```diff
@@ -272,14 +299,14 @@ def get_module_attribs(pvsam_model: pv.Pvsamv1) -> dict:
             'length':       module_length,
             'I_mp_ref':     I_mp,
             'I_sc_ref':     I_sc,
-            'P_mp_ref':     P_mp,
+            'P_mp_ref':     P_mp * 1e-3,
             'V_mp_ref':     V_mp,
             'V_oc_ref':     V_oc,
             'width':        module_width
         }
 
 
-def get_inverter_attribs(pvsam_model: pv.Pvsamv1) -> dict:
+def get_inverter_attribs(pvsam_model: pv_detailed.Pvsamv1) -> dict:
```

Comment:
Would be really nice to have a docstring here linking to the attributes in the PySAM documentation

Reply:
Done